codec-haproxy/src/main/java/io/netty/handler/codec/haproxy/HAProxyMessageDecoder.java

```diff
@@ -142,6 +155,17 @@ public HAProxyMessageDecoder() {
      * @param maxTlvSize maximum number of bytes allowed for additional data (Type-Length-Value vectors) in a v2 header
      */
     public HAProxyMessageDecoder(int maxTlvSize) {
+        this(maxTlvSize, true);
```

Comment:
default should be `false` to preserve behavoir

Reply:
The original failure strategy is fast fail